src/firefly/js/templates/fireflyviewer/FireflyViewerManager.js

```diff
@@ -129,11 +129,12 @@ function onAnyAction(layoutInfo, action, views) {
 
 function handleNewTable(layoutInfo, action) {
     const {tbl_id} = action.payload;
-    var {images={}, showImages} = layoutInfo;
+    var {images={}, showImages, showTables, hasTables} = layoutInfo;
     var {coverageLockedOn, showFits, showMeta, showCoverage, selectedTab, metaDataTableId} = images;
-
     const isMeta = isMetaDataTable(tbl_id);
-    if (isMeta || isCatalogTable(tbl_id)) {
+    const isDisplayTable = (hasTables && showTables);
```

Comment:
I don't think isDisplayTable is needed because that's what showTables is supposed to represent.  showTables should never be true unless hasTables is true.